client/my-sites/site-settings/protect.jsx

```diff
@@ -0,0 +1,187 @@
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextarea from 'components/forms/form-textarea';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	isJetpackModuleActive,
+	isJetpackModuleUnavailableInDevelopmentMode,
+	isJetpackSiteInDevelopmentMode
+} from 'state/selectors';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+
+class Protect extends Component {
+	handleAddToWhitelist = () => {
+		const { setFieldValue } = this.props;
+		let whitelist = trimEnd( this.getProtectWhitelist() );
+
+		if ( whitelist.length ) {
+			whitelist += '\n';
+		}
+
+		setFieldValue( 'jetpack_protect_global_whitelist', whitelist + this.getIpAddress() );
+	}
+
+	getIpAddress() {
+		if ( window.app && window.app.clientIp ) {
+			return window.app.clientIp;
+		}
+
+		return null;
+	}
+
+	getProtectWhitelist() {
+		const { jetpack_protect_global_whitelist } = this.props.fields;
+		return jetpack_protect_global_whitelist || '';
+	}
+
+	isIpAddressWhitelisted() {
+		const ipAddress = this.getIpAddress();
+		if ( ! ipAddress ) {
+			return false;
+		}
+
+		const whitelist = this.getProtectWhitelist().split( '\n' );
+
+		return includes( whitelist, ipAddress ) || some( whitelist, entry => {
+			if ( entry.indexOf( '-' ) < 0 ) {
+				return false;
+			}
+
+			const range = entry.split( '-' ).map( trim );
+			return includes( range, ipAddress );
+		} );
+	}
+
+	render() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			onChangeField,
+			protectModuleActive,
+			protectModuleUnavailable,
+			selectedSiteId,
+			translate
+		} = this.props;
+
+		const ipAddress = this.getIpAddress();
+		const isIpWhitelisted = this.isIpAddressWhitelisted();
+		const disabled = isRequestingSettings || isSavingSettings || protectModuleUnavailable || ! protectModuleActive;
+		const protectToggle = (
+			<JetpackModuleToggle
+				siteId={ selectedSiteId }
+				moduleSlug="protect"
+				label={ translate( 'Prevent and block malicious login attempts' ) }
+				disabled={ isRequestingSettings || isSavingSettings || protectModuleUnavailable }
+			/>
+		);
+
+		return (
+			<FoldableCard className="protect__foldable-card site-settings__foldable-card" header={ protectToggle }>
+				<QueryJetpackConnection siteId={ selectedSiteId } />
+
+				<FormFieldset>
+					<div className="protect__module-settings site-settings__child-settings">
+						<div className="protect__info-link-container site-settings__info-link-container">
+							<InfoPopover position={ 'left' }>
+								<ExternalLink href={ 'https://jetpack.com/support/protect' } target="_blank">
+									{ translate( 'Learn more about Jetpack protect' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Learn more about Jetpack. ' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).